app/components/person-search.hbs

```diff
@@ -104,6 +94,12 @@
             </td>
           </c.body>
         {{/if}}
+        {{#if (and
+          (not this.searchPeopleTask.isRunning)
+          (not this.searchPeopleTask.last.isError)
+        )}}
+          <TableMessage::SomeResult />
```

Comment:
I really like that you made components out of it, I just find `SomeResult`a weird naming, maybe `HasResults`would fit better the use case ? Or something else, but maybe something a bit more precise ?